### PR TITLE
Change colors of health checker results

### PR DESCRIPTION
**Pull request by AayushSaini101**
Related: https://github.com/jenkins-infra/plugin-site/issues/1555
SuggestedColors and Range: 
image

### CODE REVIEW

File: plugins/plugin-site/src/components/PluginHealthScoreProgressBar.jsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React, {useState} from 'react';
 import PropTypes from 'prop-types';
 import {Progress, Tooltip} from 'reactstrap';
+ import '../styles/progressbar.css';
```
Comment (halkeye) [inline plugins/plugin-site/src/components/PluginHealthScoreProgressBar.jsx:4]: can you change this to a css module, so we don't introduce global styles with generic names?

File: plugins/plugin-site/src/components/PluginHealthScoreProgressBar.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
 function PluginHealthScoreProgressBar({healthScore, name}) {
     const score = healthScore.value || 0;
     const color =
-    score > 80 ? 'success' : score > 60 ? 'warning' : 'danger';
+    score >= 80 ? "first" : score >=60 && score <80 ? 'second' : score >=40 && score <60 ? 'third'
```
Comment (halkeye) [inline plugins/plugin-site/src/components/PluginHealthScoreProgressBar.jsx:9]: I'm not sure this needs to be css classes for a single style attribute change.

**Comment by halkeye** [comment]
@alecharp

**Comment by zbynek** [comment]
The referenced issue is about the health-score tab, which is not affected by this PR. Instead this changes the color of helth bars, which are already using reasonable colors. The current scheme seems logical (resembles traffic lights), the proposed one seems random. 

@AayushSaini101 I appreciate the effort, but I currently don't see this as an improvement. Contributions that would make the health-score tab (e.g. https://plugins.jenkins.io/slack/healthscore/ ) more colorful would be appreciated though!

@halkeye @alecharp I suggest to close this.

**Comment by alecharp** [comment]
I agree we should close this.
I have some issues with hard coded values for the thresholds like that. 

There is a `statistic` object in the response the project is getting from Plugin Health Scoring could be used to have the threshold depending on the statistical scores across all the plugins (q1,q3, medium).

**Pull request closed without merging**